client/src/components/Button/Button.module.css

```diff
@@ -20,3 +20,7 @@
   padding: 5px 18px;
   font-size: 14px;
 }
+
+.btn.pill {
+  border-radius: 500px;
```

Comment:
convention is to just use `border-radius: 50%` (communicates the intended style better and also works better in the unlikely case that we ever have a button that's more than 1000px×1000px)